pkg/doc/did/doc.go

```diff
@@ -345,3 +352,73 @@ func stringEntry(entry interface{}) string {
 	}
 	return entry.(string)
 }
+
+// ToJSON converts document to raw json
+func (doc *Doc) ToJSON() ([]byte, error) {
```

Comment:
This method's name does not match what it produces. Might be clearer to rename it `RawJson`

Reply:
Out of curiosity, why raw instead of say.. Bytes?

Reply:
That works too.

Reply:
@troyronda @llorllale [Updated](https://github.com/hyperledger/aries-framework-go/compare/0dc9081c1b8917fad36c8e7230cbf4a4f67693f4..2edd9c351304abb26a71e1406933eea923d0c228#diff-ca45993820e1e4b4338acd4e485ae699R357) the name of the function to JSONBytes.